helm_lib/charts/deckhouse_lib_helm/templates/_enable_ds_eviction.tpl

```diff
@@ -0,0 +1,6 @@
+{{- /* Usage: {{ include "helm_lib_ds_eviction_annotation" . }} */ -}}
```

Comment:
```suggestion
{{- /* Usage: {{ include "helm_lib_disable_daemonset_eviction_by_cluster_autoscaler" . }} */ -}}
```
M?

Reply:
I renamed it to `helm_lib_prevent_ds_eviction_annotation` here https://github.com/deckhouse/deckhouse/pull/8609#discussion_r1661044464